db/migrate/20200529195058_add_promo_expired_to_publishers.rb

```diff
@@ -0,0 +1,5 @@
+class AddPromoExpiredToPublishers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :publishers, :promo_expiration_time, :datetime
```

Comment:
Wouldn't it be easier if we kept `promo_expiration_time` inside the `feature_flags` column?

Reply:
"expiration" might not be the best word, perhaps "lockout" is a better term e.g. "promo_lockout_time"

Reply:
https://github.com/brave-intl/publishers/pull/2733/commits/8aba6249557ebce7ad13a023acde4121cb6d127a

https://github.com/brave-intl/publishers/pull/2733/commits/179360eac96f50570ecbd2957162cfe566b24417